docs/admin-guide/migration/parallel/index.md

```diff
@@ -0,0 +1,74 @@
+# Parallel Upgrade
+This approach allows you to set up a new server environment with the updated dependencies, migrate 
+the existing database to this new environment, and apply the application upgrade to this new server. 
+The main advantage of this approach is that can perform the migration on your own timeline with 
+significantly less downtime and dependency and configuration intermingling.
+
+## Considerations
+* :white_check_mark: Reduced risk of downtime
+* :white_check_mark: Easier rollback to the old server if issues arise
+* :white_check_mark: Opportunity to test the new environment thoroughly before switching over
+* :warning: Requires more resources to maintain two environments temporarily
+* :warning: Can be more time-consuming and complex to set up
+
+## Procedure
+
+### Basic Procedure
+* [ ] Backup Database: Backup your database. Copy the backup somewhere safe.
+* [ ] Provision Resources: Provision virtual machines for the new environment.
+* [ ] Configure Server(s): Configure a new server environment with the necessary updates and dependencies.
+* [ ] Migrate Database: Migrate database backup to new server
+* [ ] Deploy Application: Deploy the updated application on the new server.
+* [ ] Verify Deployment: Check whether the server boots up properly and that there are no errors in database migrations.
+* [ ] Testing: Conduct extensive testing to ensure the application works correctly in the new environment.
+* [ ] Switch Over: Redirect traffic to the new server once testing is successful.
+* [ ] Monitoring: Monitor the new server for any issues and validate that everything is functioning as expected.
+
+## Step-by-step Instructions
+
+
+### Backup Database
+
+{% include 'admin-guide/migration/backup_database.md' %}
+
+### Setup New Server
+
+#### Provision Server(s)
+
+Use the latest Installation Guide to provision your server(s). 
+
+#### Install Dependencies
+
+Use the latest Installation Guide to install dependencies.
+
+#### Configure Server(s)
+
+Use the latest Configuration Guide to configure your server(s).
+
+### Migrate Database
```

Comment:
what about any data that was inserted since the backup was taken? Not much point to making it rolling deployable if that can't be saved. Maybe I'm misunderstanding though

Reply:
we talked about this over slack. I was assuming this "parallel" option meant rolling deploy / no downtime deploy. Really it just means installing a new instance in parallel then migrating over when you're ready.

Reply:
I think what's throwing things off for me is the "Switch Over" step is glossing over quite a lot. I think what you're suggesting is taking two backups: One at the start of the process that we can upload to the new 0.9.x instance whenever we're ready to test the instance, and then a second backup while actually switching over to the new instance while the old instance is down to capture any new data that was added since then.

So we'd want the "Disable Production Environment" step to be before we do the second backup (not sure if you're still writing all this out).

Reply:
to the point about "parallel" vs "in-place" migration, i am using those terms because those were the most common. however i still don't like "parallel" as it causes confusion. my first attempt at this was "clean slate" vs "in place", which i don't like because "clean slate" carries a conotation that is not true (i.e. we're starting fresh, including the database). but now i'm also seeing that "side-by-side" is also used to describe "parallel" migrations, so i might use that breakdown instead.

it's after struggling for

Reply:
regarding the "switch over" part, i'll just say that in general there's a lot missing so i might need some help structuring the missing buckets and maybe even writing some of these sections because my brain is mush after working on this for so long.